core/src/main/resources/springTOMCATNARAYANA.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans
+	xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation=
+		"
+			http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+		"
+	>
+
+	<import resource="springTOMCAT.xml"/>
+	<import resource="springJms.xml"/>
+
+	<bean
+		name="txManagerReal"
+		class="org.springframework.transaction.jta.JtaTransactionManager"
+		>
+		<property name="transactionManager" ref="narayanaTransactionManager"/>
+		<property name="userTransaction" ref="userTransaction"/>
+	</bean>
+
+	<bean
+		name="narayanaTransactionManager"
+		class="com.arjuna.ats.jta.TransactionManager"
+		factory-method="transactionManager"
+		depends-on="narayanaConfig,recoveryManager,defaultDataSource"
+	/>
+
+	<bean
+		name="userTransaction"
+		class="com.arjuna.ats.jta.UserTransaction"
+		factory-method="userTransaction"
+		depends-on="narayanaConfig,recoveryManager,defaultDataSource"
+	/>
+
+	<bean id="narayanaConfig" class="nl.nn.adapterframework.narayana.NarayanaConfigurationBean">
+		<property name="properties">
+			<props>
+				<prop key="JDBCEnvironmentBean.isolationLevel">2</prop> <!-- TRANSACTION_READ_COMMITTED -->
+				<prop key="CoreEnvironmentBean.nodeIdentifier">${instance.name.lc}.tm</prop>
+				<prop key="ObjectStoreEnvironmentBean.objectStoreDir">${log.dir}/${instance.name.lc}-narayana</prop>
+				<prop key="ObjectStoreEnvironmentBean.stateStore.objectStoreDir">${log.dir}/${instance.name.lc}-narayana</prop>
+				<prop key="ObjectStoreEnvironmentBean.communicationStore.objectStoreDir">${log.dir}/${instance.name.lc}-narayana</prop>
```

Comment:
De transactie logfiles zouden niet in de log.dir moeten staan (ook die van BTM niet).
Ik denk dat we daar een ${tranlog.dir} voor moeten maken.